### DISCUSSION TITLE
トレンド等に関する一意見

### DISCUSSION THREAD

**Discussion by evakichi** [increments/qiita-discussions#721] [ヘルプ]
お世話になっております。

Qiitaの「トレンド」についてお伺いします。

トレンドはどのように決定されるのでしょうか。
最近記事を拝見していても「技術的に有益な情報」が載っていないのにもかかわらずトレンド入りしている記事が散見されます。

おそらく、「いいね」を元にトレンドが作成されるのだと思いますし、プログラミングスクールなどのメンバー同士で馴れ合いで「いいね」を押してその記事がトレンドに載っているのではないかと考えています。

リンク先は明言しませんが、必要ならお知らせします。

無理だと思いますが、同じ組織からの「いいね」はカウントしないなどの措置を取っていただき、もっと有益な情報をトレンドに載せる努力をしていただきたいと思います。

そういう駄文は無視しろと言わずにこういうのを駆逐するすべを実装していただければと思います。

これをご覧になった皆様はどうお考えでしょうか。

よろしくお願いいたします。

**Comment by tanaka-asami415** [comment]
@evakichi 
ご意見ありがとうございます！

トレンドは、直近でついた「いいね」が多い順に表示しています。
https://help.qiita.com/ja/articles/qiita-feed#part-2a0ac30427d0efad

運営としても、ホームやトレンドのフィード機能について、まだまだ改善の余地があると思っています。
現時点では具体的にこれをします！と確約できませんが、ユーザーさんにとって見たい情報が沢山載っている、度々Qiitaに訪れたいと感じてもらえるような方向性に持っていきたいと考えており、今回のご意見も参考にさせていただきます🙇

**Comment by evakichi** [comment]
@tanaka-asami415 さん。

ご返答ありがとうございます。

先にも発言した通り
・同じ所属組織(Organization)からのいいね、を排除するだけでも景色が一変するのではないか
と思っているのは私だけかもしれませんが、少なくとも広告まがいの発言だけでもどうにかして排除していただきたいと思った次第です。

これからもよろしくお願いいたします。

何事もなければクローズしてください。

**Reply by tanaka-asami415** [reply]
@evakichi  さん
ご返信ありがとうございます！

先ほどの返信で一緒にご案内するのを失念しておりました🙇
現時点での代替策としては、トレンドの表示の絞り込みを行っていただくことで、個人記事とOrganization記事を分けて表示することが可能です。一度設定いただくと、フィルターの設定はログイン・未ログインに関わらず保持されます。

![image](https://github.com/increments/qiita-discussions/assets/103916185/ddc19625-0de2-4f78-b506-944be5fab6ff)

とはいえ、上記方法ですとOrganization全ての記事を見る機会が無くなってしまうため、より良くできるよう検討を続けます。

また、「[いいね](https://help.qiita.com/ja/articles/qiita-like)」は記事やコメントを読んで「いい！」「技術的に役立った」と思った時や、書いた人を「応援したい」という気持ちを表現できる機能となっており、規約違反行為となるような作為的な場合を除き、社内外を問わず気軽に賞賛し合う機能としてご活用いただきたく、同組織（Organization）内のいいねも推奨しております。

そのため、同じ所属組織(Organization)からの「いいね」を排除することは、現時点では考えておりませんが、あくまでも現時点であり、皆様のご意見も参考に検討していきたいと考えております🙇

広告を主目的としていると判断される場合は、[Qiitaコミュニティガイドライン](https://help.qiita.com/ja/articles/qiita-community-guideline#part-cf79556a40a47ce0)に沿って別途対応をしておりますので、そのような記事を見つけた際は、[通報機能](https://help.qiita.com/ja/articles/qiita-report)よりご連絡いただけますと幸いです。

こちらこそ引き続きよろしくお願いします！

**Comment by evakichi** [comment]
@tanaka-asami415 さん。

ご返答ありがとうございます。
読んでいてためになるどころかストレスになる記事が最近多い気がします。

特にプログラミングスクールと呼ばれる業者が営利目的に宣伝「させる」ケースがよく見受けられます。

このようなのを見たら報告することにしました。

今後ともよろしくお願いいたします。
